app/Voucher.php

```diff
@@ -136,11 +136,11 @@ public static function generateCodeRange($start, $end = "")
                 // We appear to be producing codes that are "0" str_pad on the left, to variable characters
                 // We'll use the $start's numeric length as the value to pad to.
                 $voucherCodes[] = $startMatch['shortcode'] . str_pad(
-                    $val,
-                    strlen($startMatch['number']),
-                    "0",
-                    STR_PAD_LEFT
-                );
+                        $val,
+                        strlen($startMatch['number']),
+                        "0",
+                        STR_PAD_LEFT
+                    );
```

Comment:
Super snarky, is this indenting wrong?

Reply:
Snark is valid!

It look like phpstorm must have brought it to my attention /  fixed it by the time we hit the l10-11 PR.
![image](https://github.com/user-attachments/assets/238e0a15-94dd-4d1e-bc3d-123b2b0f9a8a)